CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- upgraded upstream chart from 3.2.1 to 3.10.0
+- upgraded Loki from 2.6.1 to 2.7.0
+- ⚠️  Breaking changes:
```

Comment:
Can this be done like other breaking changes https://github.com/giantswarm/loki-app/blob/be8ceb10ceb6195cd6c43c7424453e7533896d5a/CHANGELOG.md?plain=1#L41 and  https://github.com/giantswarm/loki-app/blob/be8ceb10ceb6195cd6c43c7424453e7533896d5a/README.md?plain=1#L49

also, breaking change so do we go with 1.0.0 or will we only release with 4.x?

Reply:
OK, I'll move it to the README "Upgrading" section.
Concerning the next release, and its version number:
- yes, the plan is to release with 4.x, but merge in 2 steps to reduce the scope of tests/reviews.
- I don't know if next version should be 1.0.0. Good question. Until now, we are at 0.x, meaning it's not stable yet, right? That's also why we can have breaking changes between 0.4 and 0.5. I think we should wait for it to be deployed on a few of our clusters so we can better spot issues before we can declare it's stable.

Reply:
Oh yes you are right about the 0.x versions. I think customers might not be aware though :)

Reply:
- Next release will be 0.6.0
- breaking changes / upgrade notes are in the readme
- in the readme is a not saying 0.x versions are not stable and may have breaking changes between minor versions
- fixed some links